### ISSUE TITLE
Async Await Support

### ISSUE THREAD

**Issue by jelling** [firebase/firebase-functions-python#101]
Do python cloud functions support `async` and `await`? I can't find any examples showing this but it seems like a pretty basic feature. Am I missing it?

**Comment by jelling** [comment]
Figured it out. 

```
    async def blah():
        resInfer = await dosInfer()
        return resInfer

    return asyncio.run(blah())
```

**Comment by antont** [comment]
>     return asyncio.run(blah())

I think this way you loose the benefit of async, though. I mean, your code will block the instance for the whole duration of the  `run()`

I figure it would be nice to have actual async support, for a single instance to be able to run multiple IO bound functions in parallel, like the node version does I think.
